src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/simulations/ExUI.scala

```diff
@@ -27,23 +27,18 @@ class ExUI extends Simulation {
 	val caseFeederFPL = csv("CaseworkerSearchesFPL.csv").circular
 
 	// below Http Protocol is for the scenario - manage org
-  val XUIHttpProtocol = Environment.HttpProtocol
-    //.proxy(Proxy("proxyout.reform.hmcts.net", 8080).httpsPort(8080))
+  val XUIHttpProtocol = http
     .baseUrl(orgurl)
-    .headers(Environment.commonHeader)
+    .headers(Headers.navigationHeader)
 
 	//Below Http Protocol will be used for all services
 
-  val MChttpProtocol = Environment.HttpProtocol
-		//.proxy(Proxy("proxyout.reform.hmcts.net", 8080).httpsPort(8080))
-		.baseUrl(BaseURL).inferHtmlResources().silentResources
-		//.baseUrl("https://xui-webapp-perftest.service.core-compute-perftest.internal")
-		//.baseUrl("https://ccd-case-management-web-perftest.service.core-compute-perftest.internal")
+  val MChttpProtocol = http
+		.baseUrl(BaseURL)
+		.inferHtmlResources()
+		.silentResources
+    //.userAgentHeader("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36")
```

Comment:
Can we get rid of this if it's still commented out?

Reply:
I commented this out to test it. Will remove it.